packages/commonwealth/server/routes/feed.ts

```diff
@@ -0,0 +1,136 @@
+import { AppError } from '@hicommonwealth/core';
+import { Thread, ThreadAttributes, type DB } from '@hicommonwealth/model';
+import { Feed } from 'feed';
+import { slugify } from 'utils';
```

Comment:
This import breaks builds - I believe it should be `../../shared/utils`. @kurtisassad shouldn't this be caught in CI? Type-check + build are fine but as soon as you run the production build it fails.

Reply:
It's not in yet, I need reviews!
https://github.com/hicommonwealth/commonwealth/pull/6796

Reply:
This import is still breaking the build.

Reply:
@burtonator Sorry for the confusion here. Disregard my previous comment in this thread. it seems this import is breaking the build, so you will need to fix it.

Reply:
@kurtisassad how is this breaking the build? It passes all CI checks.  Is there something that is failing that is not in CI?

This is what I see on my end:

<img width="1172" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/45447/2cdcc59f-8f52-422c-91d1-b1d9621a1f58">

Reply:
I should have been more clear. It doesn't break the build (which is done in CI) it breaks the server initialization when executing `server.js`